wicket/src/runner.rs

```diff
@@ -254,6 +229,44 @@ impl RunnerCore {
     }
 }
 
+fn get_update_test_error(
+    env_var: &str,
+) -> Result<Option<UpdateTestError>, anyhow::Error> {
+    // 15 seconds should be enough to cause a timeout.
+    const DEFAULT_TEST_TIMEOUT_SECS: u64 = 15;
```

Comment:
The default progenitor client timeout is currently [15 seconds](https://github.com/oxidecomputer/progenitor/blob/6af969081a475d85e686960b917f2ee6e24251e9/progenitor-impl/src/lib.rs#L351), so maybe bump this up to 16 with a comment to that effect?

Reply:
Made it 30 seconds as a round number.